pandas/io/formats/style.py

```diff
@@ -2348,7 +2348,11 @@ def highlight_max(
         """
 
         def f(data: FrameOrSeries, props: str) -> np.ndarray:
-            return np.where(data == np.nanmax(data.to_numpy()), props, "")
+            arg = {"skipna": True}
```

Comment:
i think actually just passing the arg is better here e.g. `max(skipna=True)` as more readable

Reply:
i refactored this, pulling the function out, have a relook